### PR TITLE
[RHPAM-2264] Remove true flag for default filter

### PR DESCRIPTION
**Pull request by ruromero**
Fix https://issues.jboss.org/browse/RHPAM-2264

Cherry-picked changes for 7.4.x

Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

### CODE REVIEW

File: modules/rhpam-apb/roles/deploy-kieserver/defaults/main.yml [hunk 1]
```diff
@@ -79,15 +79,15 @@ kieserver_artifact_dir: "{{ apb_kieserver_artifact_dir | default('', true) | tri
...
+kieserver_jms_executor_transacted: "{{ apb_kieserver_jms_executor_transacted | default(false) | string | lower }}"
 kieserver_jms_queue_request: "{{ apb_kieserver_jms_queue_request | default('queue/KIE.SERVER.REQUEST', true) | trim }}"
 kieserver_jms_queue_response: "{{ apb_kieserver_jms_queue_response | default('queue/KIE.SERVER.RESPONSE', true) | trim }}"
-kieserver_jms_enable_signal: "{{ apb_kieserver_jms_enable_signal | default(false, true) | string | lower }}"
+kieserver_jms_enable_signal: "{{ apb_kieserver_jms_enable_signal | default(false) | string | lower }}"
 kieserver_jms_queue_signal: "{{ apb_kieserver_jms_queue_signal | default('queue/KIE.SERVER.SIGNAL', true) | trim }}"
-kieserver_jms_enable_audit: "{{ apb_kieserver_jms_enable_audit | default(false, true) | string | lower }}"
+kieserver_jms_enable_audit: "{{ apb_kieserver_jms_enable_audit | default(false) | string | lower }}"
 kieserver_jms_queue_audit: "{{ apb_kieserver_jms_queue_audit | default('queue/KIE.SERVER.AUDIT', true) | trim }}"
 kieserver_jms_audit_transacted: "{{ apb_kieserver_jms_audit_transacted | default(true, true) | string | lower }}"
```
Comment (errantepiphany) [inline modules/rhpam-apb/roles/deploy-kieserver/defaults/main.yml]: @ruromero Shouldn't default(true, true) here also be just default(true)? That's the way it is in your PR for master:
https://github.com/ansibleplaybookbundle/rhpam-apb/pull/98/files#diff-dbe3306deea4b36c56afdb0825f15ac0R92

**Comment by errantepiphany** [comment]
Oh, also, doesn't prepare.sh need to get rerun, because Jakub's PR https://github.com/ansibleplaybookbundle/rhpam-apb/pull/99  got merged before this one?

**Comment by ruromero** [comment]
> Oh, also, doesn't prepare.sh need to get rerun, because Jakub's PR #99 got merged before this one?

No, only I changed the apb.yml file

File: modules/rhpam-apb/roles/deploy-kieserver/defaults/main.yml (hunk 1 above)
Comment (ruromero) [reply modules/rhpam-apb/roles/deploy-kieserver/defaults/main.yml]: I didn't save the file :stuck_out_tongue_closed_eyes:

**Pull request merged**